keras/src/backend/tensorflow/core.py

```diff
@@ -696,3 +696,14 @@ def __exit__(self, *args, **kwargs):
 
 def device_scope(device_name):
     return tf.device(device_name)
+
+
+def print(*args, **kwargs):
+    """Print the specified inputs.
```

Comment:
The backend specific docstring is not visible anywhere (neither on keras.io, nor in an IDE). So if you think it is useful, move to the docstring of `keras/src/ops/core.py` explaining that this is TensorFlow specific.